build.gradle.kts

```diff
@@ -171,8 +163,16 @@ tasks {
         configureLogging()
     }
 }
+
+configurations.all {
```

Comment:
Please move this closer to the top of the script. It needs to be more visible. Perhaps, closer to other directives related to forcing/declaring the dependencies?